src/utils.ts

```diff
@@ -54,7 +54,7 @@ export async function checkModule(
   const [error, report] = await to(eslint.lintFiles(files))
 
   if (error) {
-    return Promise.reject(error)
+    throw error
```

Comment:
This is effectively the same thing as rejecting a promise within an `async` function.